core/src/storage/sd_seed_backup.py

```diff
@@ -0,0 +1,57 @@
+from micropython import const
+from typing import TYPE_CHECKING
+
+from trezor import io, utils
+from trezor.sdcard import with_filesystem
+
+if TYPE_CHECKING:
+    from typing import TypeVar, Callable
+
+    T = TypeVar("T", bound=Callable)
+
+if utils.USE_SD_CARD:
+    fatfs = io.fatfs  # global_import_cache
+
+SD_CARD_HOT_SWAPPABLE = False
+SD_SALT_LEN_BYTES = const(512)
+SD_BACKUP_PATH = "/trezor/backup"
+
+class WrongSdCard(Exception):
+    pass
+
+
+@with_filesystem
+def _load_backup(path: str) -> bytes | None:
+    # Load the mnemonic secret backup file if it exists.
+    try:
+        with fatfs.open(path, "r") as f:
+            mnemonic_secret_backup = bytearray(SD_SALT_LEN_BYTES)
+            f.read(mnemonic_secret_backup)
+
+    except fatfs.FatFSError:
+        return None
+
+
+    return mnemonic_secret_backup.decode('utf-8').rstrip('\x00').encode()
+
+
+@with_filesystem
+def load_sd_seed_backup() -> bytes | None:
+    mnemonic_secret_backup = _load_backup(SD_BACKUP_PATH)
+    if mnemonic_secret_backup is not None:
```

Comment:
isn't this a no-op?

Reply:
Copy/paste no-op, done in 2e71232f57f6a1ac4119959677f12e193b4024d3

Reply:
my point was the "if is not none -> return it -> else implicitly return None" block.

but otherwise :+1: